packages/muelu/src/Graph/MatrixTransformation/MueLu_CoalesceDropFactory_kokkos_def.hpp

```diff
@@ -188,15 +188,17 @@ namespace MueLu {
                     typename rows_type::non_const_type rows_,
                     typename cols_type::non_const_type colsAux_,
                     typename vals_type::non_const_type valsAux_,
-                    bool reuseGraph_, bool lumping_, SC /* threshold_ */) :
+                    bool reuseGraph_, bool lumping_, SC /* threshold_ */,
+                    bool filteringMayCreateDirichlet_ ) :
```

Comment:
Rename variable to `droppingMayCreateDirichlet_` to avoid confusion?

Reply:
done